### PR TITLE
*: update use of service certs ca bundle

### PR DESCRIPTION
**Pull request by squat**
cc @s-urbaniak

### CODE REVIEW

File: jsonnet/telemeter/client/kubernetes/kubernetes.libsonnet [hunk 1]
```diff
@@ -91,6 +93,8 @@ local securePort = 8443;
       local secretVolume = volume.fromSecret(secretVolumeName, secretName);
       local tlsMount = containerVolumeMount.new(tlsVolumeName, tlsMountPath);
       local tlsVolume = volume.fromSecret(tlsVolumeName, tlsSecret);
+      local sccabMount = containerVolumeMount.new(servingCertsCABundle, servingCertsCABundleMountPath);
+      local sccabVolume = volume.fromConfigMap(servingCertsCABundle, servingCertsCABundle, servingCertsCABundleFileName);
```
Comment (s-urbaniak) [inline jsonnet/telemeter/client/kubernetes/kubernetes.libsonnet]: I literally have no idea why, but my first though was "scooby doo"?

image

File: manifests/client/deployment.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ spec:
         - /usr/bin/telemeter-client
         - --id=$(ID)
         - --from=https://prometheus-k8s.openshift-monitoring.svc:9091
-        - --from-ca-file=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+        - --from-ca-file=/etc/serving-certs-ca-bundle/service-ca.crt
```
[the remark this answers is not in the document]
Comment (brancz) [reply manifests/client/deployment.yaml]: for what it's worth, prometheus reads the ca file on every scrape of a target

[the remark this answers is not in the document]
Comment (brancz) [reply manifests/client/deployment.yaml]: that's probably a good solution here too, as first scrapes may fail, but subsequent ones should start to be successful

[the remark this answers is not in the document]
Comment (squat) [reply manifests/client/deployment.yaml]: Yes good point. We definitely need to be reloading this.

**Pull request merged**